### PR TITLE
LPS-178472 Add test for adding an attachment with guest user

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/info/info-test/src/testIntegration/java/com/liferay/info/request/struts/test/AddInfoItemStrutsActionTest.java [hunk 1]
```diff
@@ -207,6 +237,14 @@ private Layout _addLayout() throws Exception {
 
 	private ObjectDefinition _addObjectDefinition() throws Exception {
 		List<ObjectField> objectFields = Arrays.asList(
+			ObjectFieldUtil.createObjectField(
```
Comment (jkappler) [inline modules/apps/info/info-test/src/testIntegration/java/com/liferay/info/request/struts/test/AddInfoItemStrutsActionTest.java:268]: @victorg1991, better use AttachmentObjectFieldBuilder here

**Review by jkappler — APPROVED** [review]
Awesome, thanks @victorg1991

**Comment by jkappler** [comment]
ci:forward

**Pull request closed without merging**